dashboards/aws-feeds/aws-feeds.yaml

```diff
@@ -0,0 +1,3042 @@
+              - date
+              - service
+              - category
+              - year
+              - month
+              - day
+          Source:
+            PhysicalTableId: 720118c4-3819-4fb9-bdbe-1666f7089c5c
+      ImportMode: SPICE
+    dependsOn:
+      views:
+      - aws_feeds_whats_new
+    schedules:
+    - default
+  aws_feeds_youtube:
+    data:
+      DataSetId: 5258b9ee-48d0-48f3-9ec4-baa4acd1d006
+      Name: aws_feeds_youtube
+      PhysicalTableMap:
+        5b308e9b-d8cf-4223-a79c-8cf1d09f371f:
+          RelationalTable:
+            DataSourceArn: ${athena_datasource_arn}
+            Catalog: AwsDataCatalog
+            Schema: ${athena_database_name}
+            Name: aws_feeds_youtube
+            InputColumns:
+            - Name: video_url
+              Type: STRING
+            - Name: title
+              Type: STRING
+            - Name: published
+              Type: STRING
+            - Name: description
+              Type: STRING
+            - Name: star_rating_count
+              Type: STRING
+            - Name: star_rating_average
+              Type: STRING
+            - Name: star_rating_min
+              Type: STRING
+            - Name: star_rating_max
+              Type: STRING
+            - Name: views
+              Type: STRING
+            - Name: year
+              Type: STRING
+            - Name: month
+              Type: STRING
+            - Name: day
+              Type: STRING
+      LogicalTableMap:
+        573c1109-b214-4bec-a018-e8847022f541:
+          Alias: aws_feeds_youtube
+          DataTransforms:
+          - CastColumnTypeOperation:
+              ColumnName: published
+              NewColumnType: DATETIME
+              Format: yyyy-MM-dd'T'HH:mm:ss'Z'
+          - CreateColumnsOperation:
+              Columns:
+              - ColumnName: youtube_embedded_url
+                ColumnId: 4e94b79b-80ea-426b-a704-63141e0bcc15
+                Expression: replace(video_url, 'https://www.youtube.com/watch?v=',
+                  'embed/')
+          - ProjectOperation:
+              ProjectedColumns:
+              - video_url
+              - title
+              - published
+              - description
+              - star_rating_count
+              - star_rating_average
+              - star_rating_min
+              - star_rating_max
+              - views
+              - year
+              - month
+              - day
+              - youtube_embedded_url
+          Source:
+            PhysicalTableId: 5b308e9b-d8cf-4223-a79c-8cf1d09f371f
+      ImportMode: SPICE
+    dependsOn:
+      views:
+      - aws_feeds_youtube
+    schedules:
+    - default
+crawlers: {}
+views:
+  aws_feeds:
+    data: |-
+      CREATE OR REPLACE VIEW ${athena_database_name}.aws_feeds AS
+      SELECT
+        date published
+      , 'blog_post' feed_type
+      , title
+      , link url
+      , category
+      FROM
+        "${athena_database_name}"."aws_feeds_blog_post"
```

Comment:
Please can you set a parameter like this : https://github.com/aws-samples/aws-cudos-framework-deployment/blob/main/dashboards/health-events/health-events.yaml#L5725

Reply:
Done